Automattic/pocket-casts-ios
# End of Year: Implements the Top Podcasts list style

| 📘 Project: #374 | 🛫 Depends on: #562 |
|:---:|:---:|

This addresses the following items:
- Updates the font sizes of all the labels
- Fixes the font size for the numbers
- Adds padding between the title and artist labels
- Fixes font weights
- Adjusts the spacings to be more inline with the designs
- Fixes the white corner appearance

This also includes the following improvements:
- The image size is now dynamically calculated based on the height of the screen which allows it to appear more consistently on smaller and larger views
- The amount of text allowed for the title and artist label is based on the height of the row which means larger screens will see more text


## Screenshots

### Design vs Implementation Comparison

<p align="center"><img src="https://user-images.githubusercontent.com/793774/205420569-57b3e266-2536-4b20-ba53-2d52ff76f534.png" width="100%" /></p>

### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
|<img src="https://user-images.githubusercontent.com/793774/205420594-9977c169-9c67-4593-b7d0-efecaa9329d2.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205420593-05f24622-260b-47fa-983c-0ac7dd1429f9.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205420592-5d671142-abea-4604-8a74-4d5a964c1c0e.png" width="320" />|


## To test

1. Launch the app
2. Tap on Profile > End of Year Card
3. Switch to the Top Podcasts story
4. ✅ Verify it looks good and matches the designs
5. ✅ Share the image and verify all the assets appear (except the logo, will be fixed)
6. Repeat the checks on various device sizes

## Checklist
